proposals/Layer-1.md

```diff
@@ -0,0 +1,390 @@
+# Layer 1 exception handling
```

Comment:
Is this meant to supersede the existing [exceptions proposal](https://github.com/WebAssembly/exception-handling/blob/master/proposals/Exceptions.md)?